openfasoc/generators/gdsfactory-gen/glayout/pdk/mappedpdk.py

```diff
@@ -14,6 +14,11 @@
 from pydantic import validate_arguments
 import xml.etree.ElementTree as ET
 
+class SpiceModels(TypedDict):
```

Comment:
I can be convinced otherwise (if you have a good reason), but keep this simple. Use a regular dictionary (do not use TypedDict) and do not make this a class. The validator function takes care that the keys are "nmos", "pmos
 or "mimcap" so we do not need more complexity than that